doc/releases/changelog-0.30.0.md

```diff
@@ -1,146 +1,285 @@
-  intermediate evolutions has increased performance as well, using the state vector ODE solver.
-  [(#4000)](https://github.com/PennyLaneAI/pennylane/pull/4000)
-  [(#4004)](https://github.com/PennyLaneAI/pennylane/pull/4004)
+  ```python
+  wires = [0, 1, 2, 3, 4, 5]
+  control_wires = wires[1:]
 
-* Added a new decomposition to `qml.SingleExcitation` that halves the number of
-  CNOTs required.
-  [(3976)](https://github.com/PennyLaneAI/pennylane/pull/3976)
+  op = qml.RX(np.pi / 2, wires=0)
 
-* Improved efficiency of `tapering()`, `tapering_hf()` and `clifford()`.
-  [(3942)](https://github.com/PennyLaneAI/pennylane/pull/3942)
+  with qml.tape.QuantumTape() as tape:
+      qml.ctrl(op, control=control_wires)
+  
+  with qml.tape.QuantumTape() as zyz_tape:
+      qml.RZ(np.pi / 2, wires=0)
+      qml.RY(np.pi / 4, wires=0)
+      qml.MultiControlledX(wires=control_wires + [0], control_values="11111", work_wires=[6, 7, 8])
+      qml.RY(-np.pi / 4, wires=0)
+      qml.MultiControlledX(wires=control_wires + [0], control_values="11111", work_wires=[6, 7, 8])
+      qml.RZ(-np.pi / 2, wires=0)
+  ```
+  
+  Fewer CNOT gates are used:
 
-* Improve the peak memory requirements of `tapering()` and `tapering_hf()` when used for larger observables.
-  [(3977)](https://github.com/PennyLaneAI/pennylane/pull/3977)
+  ```pycon
+  >>> tape.expand(depth=5).specs["gate_types"]["CNOT"]
+  60
+  >>> zyz_tape.expand(depth=5).specs["gate_types"]["CNOT"]
+  144
+  ```
+  
+  The decompositions are applied automatically when expanding tapes or decomposing operations in
+  PennyLane, but can also be accessed directly using
+  [ctrl_decomp_bisect()](https://docs.pennylane.ai/en/stable/code/api/pennylane.ops.op_math.ctrl_decomp_bisect.html).
 
-* Update Pauli arithmetic to more efficiently convert to a Hamiltonian.
-  [(#3939)](https://github.com/PennyLaneAI/pennylane/pull/3939)
+* A new decomposition to `qml.SingleExcitation` has been added that halves the number of
+  CNOTs required.
+  [(3976)](https://github.com/PennyLaneAI/pennylane/pull/3976)
+
+  ```pycon
+  >>> qml.SingleExcitation.compute_decomposition(1.23, wires=(0,1))
+  [Adjoint(T(wires=[0])),
+   Hadamard(wires=[0]),
+   S(wires=[0]),
+   Adjoint(T(wires=[1])),
+   Adjoint(S(wires=[1])),
+   Hadamard(wires=[1]),
+   CNOT(wires=[1, 0]),
+   RZ(-0.615, wires=[0]),
+   RY(0.615, wires=[1]),
+   CNOT(wires=[1, 0]),
+   Adjoint(S(wires=[0])),
+   Hadamard(wires=[0]),
+   T(wires=[0]),
+   Hadamard(wires=[1]),
+   S(wires=[1]),
+   T(wires=[1])]
+  ```
 
-* The adjoint differentiation method now supports more operations, and does no longer decompose
+* The adjoint differentiation method now supports more operations, and no longer decomposes
   some operations that may be differentiated directly. In addition, all new operations with a
-  generator are now supported by the method.
+  generator now support adjoint differentiation.
   [(#3874)](https://github.com/PennyLaneAI/pennylane/pull/3874)
```

Comment:
This is huge, there have been tonnes of user issues over this in the past. I recommend maybe highlighting this a bit more!

Reply:
oh wait is this just `default.qubit`? Or `lightning.qubit` as well? The user issues I referred to were wrt `lightning`.

Reply:
I think this is just default qubit? 🤔 The PR link is from pennylane, not pennylane-lightning... I could be wrong!

Reply:
Might be worth making it clear, I got very excited reading this lol

Reply:
The example should now make it clearer